docs/manual_tests/test_confirmation_token.md

```diff
@@ -0,0 +1,48 @@
+* Test 1 : Verification token entered is one sent on user's registered email within 24 hrs
+
+     _Screenshot/gif_: 
+     
+     ![test_1](https://user-images.githubusercontent.com/56037184/94105745-91cd9280-fe57-11ea-90e7-6e514cb3ada2.png)
+
+
+     _Expected Result_: SUCCESS (message: You have confirmed your account. Thanks!)
+
+     _Actual Result_: SUCCESS (message: You have confirmed your account. Thanks!)
+
+
+* Test 2:  Verification token of already confirmed users account entered
+
+     _Screenshot/gif_: 
+     
+     ![test_2](https://user-images.githubusercontent.com/56037184/94105790-a90c8000-fe57-11ea-8044-aee26039567e.png)
+
+
+     _Expected Result_:  SUCCESS (message: Account already confirmed)
+
+     _Actual Result_: SUCCESS (message: Account already confirmed)
+
+
+* Test 3:  Verification token of un-confirmed users account entered after 24 hrs of email being sent
+
+     _Screenshot/gif_: 
+     
+    ![test_3](https://user-images.githubusercontent.com/56037184/94105821-bf1a4080-fe57-11ea-9f9f-2d895a62fffb.png)
+
+
+     _Expected Result_:  Failure (message: The confirmation link is invalid or the token has expired.)
+
+     _Actual Result_: SUCCESS (message: You have confirmed your account. Thanks!)
+
+    _Reason_: I tested it and it is valid for 30days.
+
```

Comment:
Also, during registration, it is mentioned that token is valid for 30 days but in the confirmation, the time is mentioned as 24 hours. Please change it too so that the test passes. Make other necessary changes too if required

Reply:
According to the issue, I have supposed to just test the confirm_email token.
if there is any issue then it is mentioned in the issue to create a new issue for that.
Also, I think changing the email template in this issue is not required I can create a new issue for that and solve that issue.
What do you think @devkapilbansal @vj-codes @isabelcosta